README.md

```diff
@@ -374,6 +374,7 @@ _[IPBX](https://en.wikipedia.org/wiki/IP_PBX) telephony software_
   * [Jappix](https://jappix.com/) - Jappix is an open social platform, that let's you easily get or keep in touch with everyone. ([Source Code](https://github.com/jappix/jappix)) `AGPLv3` `PHP`
   * [Loomio](https://www.loomio.org/) - Loomio is a collaborative decision-making tool that makes it easy for anyone to participate in decisions which affect them. ([Source Code](https://github.com/loomio/loomio)) `AGPLv3` `Ruby`
   * [Mastodon](https://mastodon.social) - Federated microblogging server, an alternative to GNU social. ([Source Code](https://github.com/tootsuite/mastodon)) `AGPLv3` `Ruby`
+  * [Monica](https://monicahq.com/) - A CRM for your friends and family. ([Source Code](https://github.com/monicahq/monica)) `AGPLv3` `PHP`
```

Comment:
Please move it to [Misc/Other](https://github.com/Kickball/awesome-selfhosted#miscother) section